.github/workflows/cd.yml

```diff
@@ -0,0 +1,51 @@
+name: Business Edit UI CD
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      environement:
+        description: "Environment (dev/test)"
```

Comment:
This is a small thing but I think this is only for dev.
I believe that for test, we just tag the dev image.

Reply:
Never mind. You answered this in another PR.